src/v2/Components/ArtworkFilter/ArtworkFilters/Histogram.tsx

```diff
@@ -0,0 +1,48 @@
+import { Box, Flex } from "@artsy/palette"
+
+const BAR_WIDTH = 2
+const BAR_ROUNDNESS = 10
+
+type Range = [number, number]
+
+export interface HistogramBarEntity {
+  counts: number
+  value: number
+}
+
+interface HistogramProps {
+  selectedRange: Range
+  bars: HistogramBarEntity[]
+}
+
+const isSelected = (value: number, min: number, max: number) => {
+  return value >= min && value <= max
+}
+
+export const Histogram: React.FC<HistogramProps> = ({
+  bars,
+  selectedRange,
+}) => {
+  const maxValue = Math.max(...bars.map(bar => bar.counts))
```

Comment:
Yes, we can do this on the backend (so as not to duplicate the logic in Eigen as well)

Reply:
Actually, since this is a vanilla JS one-liner on the client side, I wonder if we can just live with the duplication for now? (I took a stab at implementing this on the backend and it got surprisingly complicated 😣)

Reply:
Yep, sure. I don't mind that